debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/helper/PrintObjectFieldsHelper.java

```diff
@@ -112,10 +112,8 @@ public static void printJ9ObjectFields(PrintStream out, int tabLevel, J9ClassPoi
 			flatObject = true;
 		}
 
-		/**
-		 * Given we have a flatObject, conditionally increment ahead to the correct object index if we have a flattened array, 
-		 * and skip ahead to the specified field by its offset
-		 */
+		/* Given we have a flatObject, conditionally increment ahead to the correct object index if we have a flattened array, 
+		 * and skip ahead to the specified field by its offset. */
```

Comment:
A comment is either all on one line, or the closer `*/` should be on a line by itself.